src/notation/internal/notationuiactions.cpp

```diff
@@ -1882,13 +1882,13 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-autoplace",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
-             TranslatableString("action", "Toggle 'automatic placement' for selected elements"),
-             TranslatableString("action", "Toggle automatic placement for selected elements")
+             TranslatableString("action", "Toggle ‘Automatic placement’ for selected elements"),
+             TranslatableString("action", "Toggle ‘automatic placement’ for selected elements")
```

Comment:
Is it intentional that the second one is without uppercase A?

Reply:
"Toggle ‘automatic placement’ for selected elements" (line 1886) and "Toggle ‘automatic placement’ (whole score)" (line 1892).

Reply:
Should be uppercase A?

Reply:
IMHO yes

Reply:
Done.